chart/templates/hpa.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.autoscaling.enabled }}
+apiVersion: autoscaling/v2beta1
```

Comment:
I think this is `autoscaling/v2beta2` now.

Reply:
Fixed